org.eclipse.help.webapp/pom.xml

```diff
@@ -25,7 +25,7 @@
       <plugin>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-jspc-maven-plugin</artifactId>
-        <version>10.0.6</version>
+        <version>10.0.14</version>
```

Comment:
shouldn't this be 10.0.15?

Reply:
https://github.com/eclipse/jetty.project/security/advisories/GHSA-p26g-97m4-6q7c

I was following this document, which says: `<=9.4.50, <=10.013 : 9.4.51, 10.0.14, 11.0.14`

Should I move up to `10.0.15`?